mailbox/api/src/main/java/org/apache/james/mailbox/StandardMailboxMetaDataComparator.java

```diff
@@ -1,66 +0,0 @@
-/****************************************************************
- * Licensed to the Apache Software Foundation (ASF) under one   *
```

Comment:
Mailbox ordering testing in LIST answer is tested by our MPT IMAP test suite.

Not ideal but enough to be safe.